app/views/episodes/_form.html.erb

```diff
@@ -12,7 +12,7 @@
           <div class="row">
             <%= render "form_main", episode: episode, form: form %>
             <%= render "form_tags", episode: episode, form: form %>
-            <%= render "form_audio", episode: episode, form: form %>
+            <%= render "form_media", episode: episode, form: form %>
```

Comment:
Lot of renaming "audio" -> "media" in these views, and moving text into `en.yml`.  And git didn't entirely pick up that they were renamed, so you'll see some deleted + new files that are really unchanged.

I'll comment the actual changes!